db-tests/src/test/java/gov/nasa/jpl/aerie/database/MerlinDatabaseTestHelper.java

```diff
@@ -120,14 +120,17 @@ int insertActivity(final int planId, final String startOffset) throws SQLExcepti
   }
 
   int insertActivity(final int planId, final String startOffset, final String arguments) throws SQLException {
+    return insertActivity(planId, startOffset, arguments, user);
+  }
+  int insertActivity(final int planId, final String startOffset, final String arguments, User modifier) throws SQLException {
```

Comment:
Suggestion, as `modifier` doesn't make sense given that this is an insert statement:
```suggestion
  int insertActivity(final int planId, final String startOffset, final String arguments, User user) throws SQLException {
```